sage-ios/SAGE/SAGE/LoginOperations.swift

```diff
@@ -184,5 +210,7 @@ class LoginOperations: NSObject {
         KeychainWrapper.removeObjectForKey(KeychainConstants.kUser)
         KeychainWrapper.removeObjectForKey(KeychainConstants.kAuthToken)
         KeychainWrapper.removeObjectForKey(KeychainConstants.kSchool)
+        KeychainWrapper.removeObjectForKey(KeychainConstants.kCurrentSemester)
+        KeychainWrapper.removeObjectForKey(KeychainConstants.kSemesterSummary)
```

Comment:
can you actually go ahead and remove the start time in the keychain if a user is mid session? the key should be in KeychainConstants file